lib/worker/src/main.rs

```diff
@@ -105,6 +108,16 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
     let pg_pool = PgPool::connect(&args.database_url).await?;
     db::setup(pg_pool.clone(), &args.database_url).await?;
 
+    let pool = pg_pool.clone();
```

Comment:
This is a very simple background routine that every minute does an update "removing" the file from cache. This will not delete the file, we'll keep the file for debugging purposes

Reply:
cool, we'll have to remember to not increase the number of workers beyond 1 in the k8 setup. ideally we'd use a https://kubernetes.io/docs/concepts/workloads/controllers/cron-jobs/

Reply:
yeah, did not want to overcomplicate something that can be ephemeral. but I don't think will be a problem, running that in multiple workers